class.jetpack-cli.php

```diff
@@ -1476,6 +1476,156 @@ private function partner_provision_error( $error ) {
 		) ) );
 		exit( 1 );
 	}
+
+	/**
+	 * Creates the essential files in Jetpack to start building a Gutenberg block or plugin.
+	 *
+	 * ## TYPES
+	 *
+	 * block: it creates a Jetpack block. All files will be created in a directory under extensions/blocks named based on the block title or a specific given slug.
+	 *
+	 * ## BLOCK TYPE OPTIONS
+	 *
+	 * The first parameter is the block title and it's not associative. Add it wrapped in quotes.
+	 * The title is also used to create the slug and the edit PHP class name. If it's something like "Logo gallery", the slug will be 'logo-gallery' and the class name will be LogoGalleryEdit.
+	 * --slug: Specific slug to identify the block that overrides the one generated based on the title.
+	 * --description: Allows to provide a text description of the block.
+	 * --keywords: Provide up to three keywords separated by comma so users  when they search for a block in the editor.
```

Comment:
I think you have an extra space here. Maybe something like this would fit better?

```suggestion
	 * --keywords: Provide up to three keywords separated by comma to help users when they search for a block in the editor.
```